### DISCUSSION TITLE
Breaking Changes for August, 2024

### DISCUSSION THREAD

**Discussion by moggy01-oss** [etsy/open-api#1276] [Q&A] [answered]
according to @nateswart  the following will be nullable in the API from August.
See: https://github.com/etsy/open-api/discussions/1260

**My question:
Is this the case for all Api users or "only" for the Turkish Api users who were already affected in the July 2024 change?
Does anyone here know more?**

see: https://github.com/etsy/open-api/discussions/1246
and see: https://github.com/etsy/open-api/releases/tag/3.0.0-general-release-2024-06-03

MESSAGE FROM @nateswart 
On August 1, 2024 we will be releasing the following breaking changes:

We will be changing a few of ShopReceipt’s fields to be nullable. These include:

first_line
second_line
city
state
zip
country_iso
The impacted endpoints are:
[createReceiptShipment](https://developer.etsy.com/documentation/reference#operation/createReceiptShipment)
[getShopReceipt](https://developer.etsy.com/documentation/reference#operation/getShopReceipt)
[getShopReceipts](https://developer.etsy.com/documentation/reference#operation/getShopReceipts)
[updateShopReceipt](https://developer.etsy.com/documentation/reference#operation/updateShopReceipt)

@turboturtleD @Gareth064

**Comment by nateswart** [comment, marked as answer]
The only change in this release is that the field itself is nullable so that we do not have to return an empty string to users without access. This release is not changing any of the rules governing who does or does not have access.
